tests/core/test_search.py

```diff
@@ -711,7 +714,7 @@ def populate_test_db(db: Session, fam_specs: Sequence[FamSpec]) -> None:
         ([_FAM_SPEC_0, _FAM_SPEC_1], 0, 10),
         # Multiple families, mixed results (some docs in FAM_2 must be missing)
         ([_FAM_SPEC_0, _FAM_SPEC_2, _FAM_SPEC_1], 0, 5),
-        # Test limit, offset
+        #  Test limit, offset
```

Comment:
Being pedantic, but Trunk.io will pick up on this extra whitespace once I've installed it in this repo

Reply:
Thanks Katy! Good spot